cfme/tests/automate/test_namespace.py

```diff
@@ -85,3 +86,35 @@ def test_duplicate_namespace_disallowed(parent_namespace):
         parent_namespace.namespaces.create(
             name=ns.name,
             description=ns.description)
+
+
+@pytest.mark.tier(2)
+def test_wrong_namespace_name(domain):
+    """To test whether namespace is creating with wrong name or not.
+       wrong_namespace: 'Dummy Namespace' (This is invalid name of Namespace because there is space
+       in the name)
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/60h
+        tags: automate
+        testSteps:
+            1. Navigate to Automation> Automate> Explorer
+            2. Try to create namespace with name `Dummy Namespace` (I put space which is invalid)
+        expectedResults:
+            1.
+            2. Should give proper flash message
+
+    Bugzilla:
+        1650071
+    """
+    wrong_namespace = 'Dummy Namespace'
+    view = navigate_to(domain.namespaces, 'Add')
+    view.name.fill(wrong_namespace)
+    view.add_button.click()
+    view.flash.assert_message('Name may contain only alphanumeric and _ . - $ characters')
+    wrong_namespace = domain.namespaces.instantiate(name=wrong_namespace)
```

Comment:
I like what you did in the previous test with the finalizer, we should put that here as well. If that namespace does get created, we want to clean it up.